person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def initialize(age, name = 'Unkown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @parent_permission = parent_permission
+    @age = age
+  end
+
+  def can_use_services?
+    age >= 18 || parent_permission == true
+  end
+
+  private
+
+  def of_age?
```

Comment:
- [ ] The purpose of defining the `of_age?` method is to simply check if a person's age is greater than 18. Writing `age >= 18` in `can_use_services?` makes **of_age?** useless and breaks the DRY rule. Kindly replace `age >= 18` with `of_age?` on line 13.

- [ ] Let's use the instance variable we defined. On line 13, instead of `parent_permission == true`, I would suggest you use `@parent_permission` to make your code look cleaner.